### PR TITLE
RandomARQ

### PR DESCRIPTION
**Pull request by ArqTras**
https://github.com/arqma/RandomARQ

### CODE REVIEW

File: src/crypto/randomx/randomx.cpp [hunk 1]
```diff
@@ -70,6 +70,36 @@ RandomX_ConfigurationWownero::RandomX_ConfigurationWownero()
...
+{
+	ArgonSalt = "RandomARQ\x01";
+	ProgramIterations = 1024;
+	ProgramSize = 256;
+	ProgramCount = 4;
+	ArgonIterations = 1;
+	ArgonLanes = 1;
+	ScratchpadL2_Size = 131072;
+	ScratchpadL3_Size = 262144;
+
```
Comment (SChernykh) [inline src/crypto/randomx/randomx.cpp]: You only need to assign values that are different from the default (Monero) config. As far as I can see, it's RANDOMX_ARGON_SALT, RANDOMX_PROGRAM_ITERATIONS, RANDOMX_PROGRAM_COUNT, RANDOMX_SCRATCHPAD_L3, RANDOMX_SCRATCHPAD_L2. Everything else is the same and doesn't need to be in this constructor.

**Comment by xmrig** [comment]
@ArqTras Pull request to official xmrig repository are welcome (evo branch with OpenCL backend) preferable with single commit and other information: testnet pool, estimated fork date, etc.
Thank you.

**Comment by SChernykh** [comment]
@ArqTras Just post a test pool url here, I can create a PR to the main xmrig repository today as soon as I can test it.

**Comment by ArqTras** [comment]
@SChernykh http://stagenet.supportcryptonight.com/
address to connect mm.supportcryptonight.com:3333
sample wallet : as2iuspY683LiNdLEy9nSg7xtoJUP3QwFhgJMprp4VoYMvANcsnFLeY7UvgNf2cPMXBHf6EuYeS2UjHaesjhu1cK2xTPhbyhB

We do not decide yet about fork date but it will be soon announced

**Comment by SChernykh** [comment]
@ArqTras Thanks, I'll look into it this evening.

**Comment by SChernykh** [comment]
https://github.com/xmrig/xmrig/pull/1227

**Comment by ArqTras** [comment]
@SChernykh Thank You

**Comment by ArqTras** [comment]
Fork date proposed is 8.11.2019
Fork block 303666
Epoch timestamp: 1573257000
Date and time (GMT): Friday, 8 November 2019 23:50:00

**Comment by MoneroOcean** [comment]
Took rx/arq support from original xmrig repo, so no additional PR changes are required here. Thank you!

**Pull request closed without merging**